### PR TITLE
Spanish version of guides docs

### PR DESCRIPTION
**Pull request by tatang26**
Hello (more Spanish content 🇪🇸)

This PR includes the spanish translation of **almost** all of docs from **Guides** section.

I adjusted some code examples and fixed some tags that were not rendering well.

_The excluded sections are:_
- Plugins
- Local Authentication
- Third Party Authentication

I'll translate them when issue [680](https://github.com/gobuffalo/docs/issues/680) is fixed!

### CODE REVIEW

**Review by paganotoni — DISMISSED** [review]

File: content/documentation/guides/localization.es.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+### Crear versiones con sufijos de las plantillas
+
+Primero, crea una versión para la configuración regional por defecto, sin un sufijo de lenguaje:
+
+**page.html**:
+```html
+<p>This is my default language page.</p>
+```
+
+Then, create a new suffixed version for each language you want to support:
```
Comment (YanDeLeon) [inline content/documentation/guides/localization.es.md:108]: Can you please translate this ?

File: content/documentation/guides/events.es.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+"myapp:foo:start"
+"myapp:foo:stop"
+"mypkg:workers:bar:err"
+```
+
+Este patrón de nomenclatura facilita a los usuarios filtrar solo aquellos eventos que les interesan. Ve a [Filtrando Eventos](#filtering-events) para mas detalles.
+
+---
+
+Hay varias maneras de emitir un {{< doclink message="events#Event" href="github.com/gobuffalo/events#Event" >}} en tu código de Go. LAs funciones {{< doclink message="events#EmitError" href="github.com/gobuffalo/events#EmitError" >}} y {{< doclink message="events#EmitPayload" href="github.com/gobuffalo/events#EmitPayload" >}} aceptan un argumento de tipo `payload interface{}`. Se recomienda usar {{< doclink message="events#Payload" href="github.com/gobuffalo/events#Payload" >}} para datos de entrada; cualquier otro tipo pasado será convertido a {{< doclink message="events#Payload" href="github.com/gobuffalo/events#Payload" >}} con el argumento establecido en los datos de entrada con la llave `data`.
```
Comment (YanDeLeon) [inline content/documentation/guides/events.es.md:48]: In "LAs", two first letters are capitalized

**Review by Kevindm14 — DISMISSED** [review]

File: content/documentation/guides/localization.es.md (hunk 1 above)
Comment (YanDeLeon) [inline content/documentation/guides/localization.es.md:108]: Can you please translate this ?

File: content/documentation/guides/events.es.md (hunk 2 above)
Comment (YanDeLeon) [inline content/documentation/guides/events.es.md:48]: In "LAs", two first letters are capitalized

**Review by YanDeLeon — CHANGES_REQUESTED** [review]
Left some comments on events and localization files

File: content/documentation/guides/events.es.md (hunk 2 above)
Comment (tatang26) [reply content/documentation/guides/events.es.md]: Good catch @YanDeLeon , fixing

**Review by YanDeLeon — APPROVED** [review]

**Pull request merged**